README.md

```diff
@@ -69,12 +68,31 @@ cd integreatly-operator
 
 If you are working against a fresh cluster it will need to be prepared using the following. 
 Ensure you are logged into a cluster by `oc whoami`.
-Include the `INSTALLATION_TYPE`. See [here](#variables-table) about this and other optional configuration variables.
+Include the `INSTALLATION_TYPE`. See [here](#3-configuration-optional) about this and other optional configuration variables.
 ```shell
 INSTALLATION_TYPE=<managed/managed-api> make cluster/prepare/local
 ```
 
-### 3. Run integreatly-operator
+
+### 3. Configuration (optional)
+
+If you are running RHOAM against a cluster which is smaller than the requirements listed above, you 
+should use the IN_PROW variable, otherwise the installation will not complete. 
+If you have a cluster which meets the requirements, this step can be skipped.
+Please see the table below for other configuration options.
+
+```shell script
+INSTALLATION_TYPE=managed-api IN_PROW=true USE_CLUSTER_STORAGE=<true/false> make deploy/integreatly-rhmi-cr.yml
```

Comment:
It looks.... rather feels strange now. Hear (read?) me out. Before with just one variable, it was more like an example. Now with all 3, it looks like a suggestion. INSTALLATION_TYPE and IN_PROW with only one "option" can be interpreted as "we recommend to do so" while USE_CLUSTER_STORAGE with `<.../...>` could be "take a look at the table and chose". But I'm not even sure this is a thing - could be too paranoid.

Reply:
Yes, For this case IN_PROW is mostly used for RHOAM(managed-api) situation. I'm not even sure if it can be used in the case of RHMI. 🤔  so it was intentional that i didn't give the two options here but was really specific about what should be run. In the first instance I left out USE_CLUSTER_STORAGE but when I moved the table down I decided to include it but in the small cluster case it doesn't matter really what the user choses for that value.